test/msw-api/setup-worker/start/on-unhandled-request/warn.test.ts

```diff
@@ -26,10 +26,16 @@ test('warns on an unhandled request when using the "warn" option', async () => {
   expect(status).toBe(404)
 
   const libraryErrors = messages.error.filter(filterLibraryLogs)
-  const libraryWarnings = messages.warning.filter(filterLibraryLogs)
+  const [libraryWarning] = messages.warning.filter(filterLibraryLogs)
```

Comment:
This assumes the first warning will be related to the unhandled request. That is not a safe assumption, as there is no guarantee of an exact warning index. 

Instead, access all warnings and find the one you are attempting to assert (how it was previously).

Reply:
Noted. Having the same issue I mentioned above regarding getting formatting to match.

Reply:
Disregard, formatting isn't an issue when the output is right :)